Refresh.GameServer/Database/GameDatabaseContext.Levels.cs

```diff
@@ -244,25 +252,28 @@ public DatabaseList<GameLevel> GetHighestRatedLevels(int count, int skip, TokenG
             .Select(x => x.Level)
             .Where(l => l != null)
             .Where(l => l._Source == (int)GameLevelSource.User)
+            .FilterByLevelFilterSettings(user, levelFilterSettings)
             .FilterByGameVersion(gameVersion);
 
         return new DatabaseList<GameLevel>(highestRated, skip, count);
     }
     
     [Pure]
-    public DatabaseList<GameLevel> GetTeamPickedLevels(int count, int skip, TokenGame gameVersion) =>
+    public DatabaseList<GameLevel> GetTeamPickedLevels(int count, int skip, TokenGame gameVersion, GameUser? user, LevelFilterSettings levelFilterSettings) =>
         new(this.GetLevelsByGameVersion(gameVersion)
             .Where(l => l.TeamPicked)
+            .FilterByLevelFilterSettings(user, levelFilterSettings)
             .OrderByDescending(l => l.PublishDate), skip, count);
 
     [Pure]
-    public DatabaseList<GameLevel> GetDeveloperLevels(int count, int skip, TokenGame gameVersion) =>
+    public DatabaseList<GameLevel> GetDeveloperLevels(int count, int skip, GameUser? user, LevelFilterSettings levelFilterSettings) =>
         new(this._realm.All<GameLevel>()
             .Where(l => l._Source == (int)GameLevelSource.Story)
+            .FilterByLevelFilterSettings(user, levelFilterSettings)
```

Comment:
We can probably skip passing the user in here since a developer level will never ever be owned by a user.